src/components/admin/article/articleLogic.js

```diff
@@ -0,0 +1,140 @@
+import Plain from 'slate-plain-serializer';
+import _ from 'lodash';
+import { parseFalcorPseudoArray } from 'lib/falcor/falcor-utilities';
+
+export async function validateChanges(
+  articleId,
+  componentState,
+  displayConfirm,
+) {
+  const falcorData = componentState.data.articles.byId[articleId];
+
+  let processedAuthors = componentState.authors.map(author => author.id);
+
+  // Check that all authors are unique
+  if (_.uniq(processedAuthors).length !== processedAuthors.length) {
+    return {
+      invalid: true,
+      msg:
+        "You have duplicate authors, as this shouldn't be able" +
+        ' to happen, please contact developers. And if you know all the actions' +
+        ' you did previously to this and can reproduce them that would be of' +
+        ' great help. The save has been cancelled',
+    };
+  }
+  // Check that there wasn't already authors and they deleted all of them
+  if (
+    processedAuthors.length === 0 &&
+    (falcorData.authors &&
```

Comment:
It's these 2 lines I actually added, otherwise I only refactored. This is what fixed the bug